File: CS/WpfApplication1/MyChartControl/AnotationPresenter.cs
using System;
using System.Linq;
using System.Windows;
using System.Windows.Controls;
using System.Collections.Generic;

namespace MyChartControl
{
    public class AnotationPresenter : Control
    {
        static AnotationPresenter()
        {
            DefaultStyleKeyProperty.OverrideMetadata(typeof(AnotationPresenter),
                     new FrameworkPropertyMetadata(typeof(AnotationPresenter)));
        }

        public AnotationPresenter()
        {

        }
    }
}

File: VB/WpfApplication1/MyChartControl/AnotationPresenter.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Linq
Imports System.Windows
Imports System.Windows.Controls
Imports System.Collections.Generic

Namespace MyChartControl
	Public Class AnotationPresenter
		Inherits Control
		Shared Sub New()
			DefaultStyleKeyProperty.OverrideMetadata(GetType(AnotationPresenter), New FrameworkPropertyMetadata(GetType(AnotationPresenter)))
		End Sub

		Public Sub New()

		End Sub
	End Class
End Namespace
